hack/make-rules/test.sh

```diff
@@ -155,7 +151,14 @@ for arg; do
   fi
 done
 if [[ ${#testcases[@]} -eq 0 ]]; then
-  kube::util::read-array testcases < <(kube::test::find_dirs)
+  normalize="false"
+  kube::util::read-array testcases < <(kube::test::find_packages)
+fi
+
+if [[ "${normalize}" == "true" ]]; then
```

Comment:
why a bool instead of an `else`?

Reply:
Good point - it evolved and I missed this cleanup.